packages/cli/package.json

```diff
@@ -47,6 +47,7 @@
     "@vercel/hono": "0.2.14",
     "@vercel/hydrogen": "1.3.2",
     "@vercel/next": "4.15.6",
+    "@vercel/nestjs": "workspace:*",
```

Comment:
Shouldn't this point to the version number instead like the other builders?

Reply:
This should get converted to the real version on release, I think we do the others incorrectly actually, but let me double check.